### PR TITLE
Enable Dependabot

### PR DESCRIPTION
**Pull request by batmat**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+- package-ecosystem: maven
+  directory: "/"
+  schedule:
+    interval: daily
```
Comment (MRamonLeon) [inline .github/dependabot.yml:6]: do you consider weekly?

Comment (batmat) [reply .github/dependabot.yml:6]: I think daily makes things more continuous, rather than taking a batch of things regularly?

Comment (MRamonLeon) [reply .github/dependabot.yml:6]: My reasoning is weekly is frequent enough and may save us from getting a bad release.

Comment (batmat) [reply .github/dependabot.yml:6]: I don't mind that much. Up to you :)

**Review by oleg-nenashev — APPROVED** [review]
Weekly updates would be more than enough. There are not that many moving parts in this plugin anyway

**Review by MRamonLeon — APPROVED** [review]

**Pull request merged**